.github/workflows/run-tests.yml

```diff
@@ -34,16 +40,43 @@ jobs:
         python-version: [3.11]
 
     steps:
+      - name: Eco CI Energy Estimation - Initialize
+        uses: green-coding-solutions/eco-ci-energy-estimation@main
+        with:
+          task: start-measurement
+          company-uuid: "7f1d34f0-7666-4378-8d8f-d37cced7ccb0"
+          project-uuid: "a9946d61-149b-4757-99e5-aaac24acf289"
+        continue-on-error: true
+
       - uses: actions/checkout@v4
+      - name: Eco CI Energy Estimation - Get Measurement
```

Comment:
This step extra step is measuring the checkout stage. If we wanted to measure other steps in the build process we'd add the `green-coding-solutions/eco-ci-energy-estimation@main`, but presumably once we have the project and org id set, we needn't repeat it in later steps